prisma/migrations/20221002205940_add_excercise_and_event_type_enum/migration.sql

```diff
@@ -0,0 +1,12 @@
+/*
+  Warnings:
+
+  - Added the required column `updatedAt` to the `Exercise` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterEnum
+ALTER TYPE "EventType" ADD VALUE 'EXERCISE';
+
+-- AlterTable
+ALTER TABLE "Exercise" ADD COLUMN     "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+ADD COLUMN     "updatedAt" TIMESTAMP(3) NOT NULL;
```

Comment:
This migration will crash the production database. Please delete this file, add a default to the `updatedAt` column on the Exercise model of `@default(now())` and generate a new migration

Reply:
Updated

Reply:
updated